src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -665,9 +665,11 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                 return;
             }
 
-            const auto modifiers = args.KeyModifiers();
-            const auto altEnabled = WI_IsFlagSet(modifiers, VirtualKeyModifiers::Menu);
-            const auto shiftEnabled = WI_IsFlagSet(modifiers, VirtualKeyModifiers::Shift);
+            const auto modifiers = static_cast<uint32_t>(args.KeyModifiers());
+            // static_cast to a uint32_t because we can't use the WI_IsFlagSet
+            // macro directly with a VirtualKeyModifiers
+            const auto altEnabled = WI_IsFlagSet(modifiers, static_cast<uint32_t>(VirtualKeyModifiers::Menu));
+            const auto shiftEnabled = WI_IsFlagSet(modifiers, static_cast<uint32_t>(VirtualKeyModifiers::Shift));

```

Comment:
worried about unrelated code cahnges

Reply:
Ah this was because of a cppwinrt update. We can't just `WI_IsFlagSet` the `VirtualKeyModifiers` enum anymore.

Reply:
#bydesign